R/format_output.R

```diff
@@ -0,0 +1,87 @@
+#' Neatly format CFR estimates
+#'
+#' @description Format the CFR MLE and 95% range estimates neatly, giving the
+#' estimates to 3 significant figures and presenting the MLE and 95% interval
+#' on a single line
+#'
+#' @param df_in A data.frame of the format outputted by [estimate_static()]
+#' or [estimate_time_varying()].
+#'
+#' @param estimate_type A required string describing whether a severity of
+#' reporting estimate is to be formatted. The package calculates both severity
+#' and under-ascertainment estimates and uses the same formatting function for
+#' both
+#'
+#' @param type An optional string describing the estimate, typically used to
+#' describe whether the estimate has been corrected for delays or not
+#'
+#' @return Prints the output of [estimate_static()] or [estimate_time_varying()]
+#' neatly to the R terminal.
+#'
+#' @export
+#'
+#' @examples
+#' # load Ebola 1976 outbreak data
+#' data("ebola1976")
+#'
+#' # add the location the dataset relates to, for ease of formatting at the end
+#' # of this example
+#' ebola1976$location <- "Democratic Republic of the Congo"
+#'
+#' # subset the data so we focus on the earlier half of the outbreak
+#' df_ebola_subset <- subset(ebola1976, date <= "1976-09-30")
+#'
+#' # read epidist for EVD onset to death from {epiparameter}
+#' onset_to_death_ebola <- epiparameter::epidist_db(
+#'   disease = "Ebola Virus Disease",
+#'   epi_dist = "onset_to_death",
+#'   author = "Barry_etal"
+#' )
+#'
+#' # Calculate static naive CFR
+#' ncfr <- estimate_static(
+#'   df_in = df_ebola_subset,
+#'   correct_for_delays = FALSE,
+#'   group_by = "location"
+#' )
+#'
+#' # Calculate static corrected CFRs
+#' ccfr <- estimate_static(
+#'   df_in = ebola1976,
+#'   correct_for_delays = TRUE,
+#'   epi_dist = onset_to_death_ebola,
+#'   group_by = "location"
+#' )
+#'
+#' # Formats the output of the CFR data.frames nicely
+#' # and prints to the terminal
+#' format_output(ncfr, estimate_type = "severity", type = "Naive")
+#' format_output(ccfr, estimate_type = "severity", type = "Corrected")
+#'
+format_output <- function(df_in,
+                          estimate_type,
+                          type = NULL) {
```

Comment:
Is this function really necessary? Could we not have stuck to the earlier implementation of the severity estimate as a named vector with three values? That handles the pretty printing issue to some extent.